### ISSUE TITLE
multiple downlinks

### ISSUE THREAD

**Issue by parylizer** [chirpstack/chirpstack-packet-multiplexer#30]
Is it possible to have multiple downlinks with this? 

For example: two packet forwarders --> multiplexer --> two backend servers and then downlinks will return back to both of the packet forwarders

**Comment by brocaar** [comment]
I'm not sure if I understand the question. But you could have multiple gateways connecting to one multiplexer, which then sends the data to multiple LNS instances (e.g. ChirpStack, TTN, .....). Unless configured for uplink only, each LNS will be able to send downlink as if the gateway was directly connected.

**Comment by parylizer** [comment]
> I'm not sure if I understand the question. But you could have multiple gateways connecting to one multiplexer, which then sends the data to multiple LNS instances (e.g. ChirpStack, TTN, .....). Unless configured for uplink only, each LNS will be able to send downlink as if the gateway was directly connected.

Yes, that sounds like what I'm looking for except I should clarify that I would like the downlink to be sent to all gateways either with the same gateway id or if it can send to multiple with different id's (preferred). When tested previously it seemed like the downlink would only go to one gateway.

**Comment by brocaar** [comment]
> I would like the downlink to be sent to all gateways either with the same gateway id or if it can send to multiple with different id's (preferred).

A downlink is always set to a single gateway, because it is targeted at a specific Gateway ID. As well, for Class-A downlinks, the downlink message contains a `tmst` value that is specific to the targeted gateway.

Depending your needs, you might be able to implement an other solution, but this is not an use-case covered by the Multiplexer. The use-case of the multiplexer is to share one or multiple gateways with multiple backends and let each backend think each gateway is directly connected.
